example/layer-examples.js

```diff
@@ -68,6 +68,8 @@ const ScatterplotLayerExample = props =>
   new ScatterplotLayer({
     data: props.points,
     opacity: 0.5,
+    outlineMode: true,
+    outlineStrokeWidth: 2,
```

Comment:
I think to make it consistent with choropleth (`drawContour`) the `outlineMode` option should be `drawOutline`